docs/guide/index.asciidoc

```diff
@@ -7,22 +7,18 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 [[overview]]
 == Overview
 
-APM is an application performance monitoring system built on the Elastic Stack.
-It uses Elasticsearch as its data store and allows you to monitor software services and applications in real time.
-Regardless of whether your code is best described as a service or as an application,
-this guide will indiscriminately use the word service.
-
-With APM you can automatically collect detailed performance information from inside your service with only minor changes to your code.
-For example APM agents can measure the response time for incoming requests,
-time spent in database queries,
+Elastic APM is an application performance monitoring system built on the Elastic Stack.
+It allows you to monitor software services and applications in real time (this guide will indiscriminately use the word service),
```

Comment:
The parenthetical statement here disrupts the flow of this sentence too much. Maybe add it as a note instead of parenthetically?